config/pmd-ruleset.xml

```diff
@@ -64,7 +59,6 @@
         <exclude name="TooManyStaticImports" />
         <exclude name="UseUnderscoresInNumericLiterals" />
         <exclude name="LinguisticNaming" />
-        <exclude name="UnnecessaryFullyQualifiedName" />
```

Comment:
I'd actually prefer to keep excluding this rule (and just remove the commit) if it's all the same to you? I find it can sometimes really help readability to be able to qualify static calls.